test_tools/sigma_verify_rules.py

```diff
@@ -104,9 +159,12 @@ def move_problematic_rule(filepath, move_to_path, reason=None):
             file_objec.write(f'{filepath}\n{reason}\n\n')
 
         base_path = os.path.basename(filepath)
+        logging.info('Moving the rule: {0:s} to {1:s}'.format(
+        filepath, f'{move_to_path}{base_path}'))
```

Comment:
indentation is weird here

Reply:
but you didn't fix the indentation it's still not correct

```python
logging.info('Moving the rule: {0:s} to {1:s}'.format(
    filepath, ...
```

I'm still seeing:
```python
logging.info('Moving the rule: {0:s} to {1:s}'.format(
filepath, ...
```